snapcraft/tests/test_repo.py

```diff
@@ -60,6 +60,21 @@ def _fetch_binary(download_dir, **kwargs):
         self.mock_cache.return_value.get_changes.return_value = [
             self.mock_package]
 
+    def test__get_pkg_name_parts_name_only(self):
+        name, version = repo._get_pkg_name_parts('hello')
+        self.assertEqual('hello', name)
+        self.assertEqual(None, version)
+
+    def test__get_pkg_name_parts_all(self):
```

Comment:
I've always done that then the method I'm testing starts with an underscore.  More as a mental note than anything.  If you want me to make it a single quote I'm happy to do so.